README.md

```diff
@@ -58,16 +64,17 @@ cargo install --path .
 
 ### How to use it?
 
-diffr tries to be a well behaved unix program: it reads its input from stdin
+diffr tries to be a well behaved Unix program: it reads its input from stdin
 and writes to stdout.
 
-#### To test it:
+#### One-off usage
+
 ```
 cargo build --release
```

Comment:
Should this line be here, @mookid? How does it interact with the the section above about building from source?

Reply:
It was meant to be closer to the top of the readme, and predates the installation instructions. You can indeed remove that section.

Reply:
> You can indeed remove that section.

Not sure I follow. It seems strange that you'd suggest removing the entire [Installation section](https://github.com/mookid/diffr#installation). Did you perhaps mean remove the `cargo build --release` line?

Reply:
yes, I am talking about thse (now misplaced) lines.

Reply:
Ok, I've submitted #54 to remove that line.